### PR TITLE
Add correct path to run NAS ref locally in README

### PR DESCRIPTION
**Pull request by eoin-shaughnessy-cko**
# Description of changes in PR

[//]: # 'Please add your description here'

## Checklist

- [ ] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +53,3 @@ To run this project locally, clone the repository and run these commands in sequ
 3. `npm run start`
 
 The project should then be available at port 3001 (navigate to `localhost:3001` in your browser).
-
-For example:
-
-`localhost:3001` (ABC)
-`localhost:3001/preview/crusoe` (NAS)
```
Comment (eoin-shaughnessy-cko) [inline README.md:60]: Lines 56-60 felt redundant since ABC and NAS refs now use the same thing.

Comment (natasha-moore-cko) [reply README.md:60]: if ABC & NAS are the same URLs, how would we differentiate the two when running them locally?
I think the ABC one is now under http://localhost:3001/previous/ (at least that URL works for me)

Comment (eoin-shaughnessy-cko) [reply README.md:60]: Would this be more accurate?

> For example:
> 
> `localhost:3001` (ABC)
> `localhost:3001/previous` (NAS)

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The API reference is hosted on Vercel.
 
 ## NAS
 
-Append `/preview/crusoe` to all links (including preview ones) for the NAS API reference.
+For both the NAS and ABC API references, append `/localhost:3001` to all links (including preview ones).
```
Comment (natasha-moore-cko) [inline README.md:15]:
```suggestion
Append `/previous` to all links (including preview ones) for the ABC API reference.
```

File: README.md (hunk 1 above)
Comment (natasha-moore-cko) [reply README.md:60]: Yep, looks correct

**Review by natasha-moore-cko — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ The API reference is hosted on Vercel.
 
 ## NAS
```
Comment (natasha-moore-cko) [inline README.md:13]:
```suggestion
## ABC
```

**Review by natasha-moore-cko — APPROVED** [review]

**Pull request merged**